packages/workspace/src/utils/community-plugins.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+
+const APPROVED_PLUGINS_JSON_URL =
+  'https://raw.githubusercontent.com/nrwl/nx/master/community/approved-plugins.json';
+
+export interface CommunityPlugin {
+  name: string;
+  url: string;
+  description: string;
+}
+
+export async function getCommunityPlugins(): Promise<CommunityPlugin[]> {
+  let plugins = [];
+
+  try {
+    const response = await axios.get<CommunityPlugin[]>(
+      APPROVED_PLUGINS_JSON_URL
+    );
+    plugins = response.data;
+  } catch (error) {
+    console.log(error);
```

Comment:
use output.warn instead

Reply:
or maybe combine with the output.log above.

I think the flow is a bit confusing. You are printing an error and then return an empty array that gets ignored. I think it might be better to either print the list or print a warning and not worry about the empty array

Reply:
OK I will clean it up some and just have a `listCommunityPlugins` method that handles the fetching and displaying. Should clean up the flow.

Reply:
OK I cleaned up the flow a bit with a `listCommunityPlugins` method that either outputs the plugins or a warning. Let me know what you think.